submissions/sofiichuk/pop-up-task/index.html

```diff
@@ -0,0 +1,50 @@
+    <title>dropdown</title>
+    <meta charset="UTF-8">
+</head>
+<body>
+<nav>
+    <ul class="navBar">
+        <li class="navBar__item item_txt"><a href="#">Mail</a></li>
+        <li class="navBar__item item_txt"><a href="#">Images</a></li>
+        <li class="navBar__item item_btn">
+            <input id="input_1" type="checkbox">
+            <label for="input_1" id="label_1">
+                <img alt="dropdown menu" src="icons/dropdown_menu.png">
+            </label>
+                <ul class="dropdownMenu">
+                    <li><a href="#"><img alt="" src="icons/calendar.png"/><span>Calendar</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/mail.png"/><span>Mail</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/news.png"/><span>News</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/maps.png"/><span>Map</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/files.png"/><span>Files</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/contacts.png"/><span>Contacts</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/translation.png"/><span>Translate</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/videos.png"/><span>Videos</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/search.png"/><span>Search</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/collection.png"/><span>Collection</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/apps.png"/><span>Apps</span></a></li>
+                    <li><a href="#"><img alt="" src="icons/chat.png"/><span>Chat</span></a></li>
+                    <li class="showMoreButton">
+                        <input id="input_2" type="checkbox">
+                        <label id="label_2" for="input_2"></label>
+                        <ul class="dropdownMenu__collapsiblePart">
+                            <li><a href="#"><img alt="" src="icons/documents.png"/><span>Document</span></a></li>
+                            <li><a href="#"><img alt="" src="icons/classroom.png"/><span>Classroom</span></a></li>
+                            <li><a href="#"><img alt="" src="icons/photos.png"/><span>Photos</span></a></li>
+                            <li><a href="#"><img alt="" src="icons/mars.png"/><span>Mars</span></a></li>
+                            <li><a href="#"><img alt="" src="icons/blogs.png"/><span>Blogs</span></a></li>
+                            <li><a href="#"><img alt="" src="icons/promo.png"/><span>Promos</span></a></li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+        <li class="navBar__item item_img"><a href="#"><img alt="notifications" src="icons/bell.png"/></a></li>
+        <li class="navBar__item item_img"><a href="#"><img alt="account" src="icons/user.png"/></a></li>
```

Comment:
For icons inside the link in the popup, you set an empty value, but for the rest icons, you leave the populated `alt` attribute, so what about them?

Reply:
I removed alt tags where they were duplicated with span tags (so that screenreaders wouldn't say: "videos videos"). In another few cases - in navbar - there's no span tags - just icons. If I remove them, blind people would hear nothig, i guess.

Reply:
Generally for that case on interactive elements adds `aria-label` attribute.

Reply:
Could I leave these alt tags?